content/collections/guides_and_surveys/en/proxy.md

```diff
@@ -0,0 +1,83 @@
+---
+id: 9efa3729-a771-4659-ab24-710758c53755
+title: 'Proxy requests to G&S with AWS CloudFront'
+exclude_from_sitemap: false
+---
+
+Set up a single CloudFront distribution to reverse proxy both static assets and G&S API traffic. This can help circumvent domain blocking in certain regions or by specific extensions and DNS servers. Because G&S APIs and static assets are latency-sensitive, Amplitude recommends using edge-hosted solutions to minimize round-trip time.
```

Comment:
```suggestion
title: 'Proxy requests to Guides and Surveys with AWS CloudFront'
exclude_from_sitemap: false
---

Set up a single CloudFront distribution to reverse proxy both static assets and Guides and Surveys API traffic. This can help circumvent domain blocking in certain regions or by specific extensions and DNS servers. Because Guides and Surveys APIs and static assets are latency-sensitive, Amplitude recommends using edge-hosted solutions to minimize round-trip time.
```

Reply:
there are some official guidelines out there somewhere to always use `Guides and Surveys`